db/indextest/indextest_test.go

```diff
@@ -249,9 +259,10 @@ func TestInitializeIndexes(t *testing.T) {
 		collections bool
 	}{
 		{true, false},
-		{false, false},
+		/*{false, false},
```

Comment:
Are these test cases meant to be commented out?

Reply:
nope, fixed